lib/view/icu_non_intensivist/ventilation/ventilation_view.dart

```diff
@@ -17,24 +20,39 @@ class VentilationView extends StatelessWidget {
     // Analytics set screen name, stays until another screen changes it
     Analytics.analyticsScreen(Constants.analyticsVentilationScreen);
 
-    final tabs = ventilationGuide.map((e) => e.heading).toList();
-
-    return TabViewTemplate(
-      Strings.ventilationTitle,
-      color: AppColors.blue50,
-      tabs: tabs,
-      actions: <Widget>[
-        IconButton(
-          icon: SvgPicture.asset(
-            'assets/images/icon/SVG/icon_infographic.svg',
-          ),
-          onPressed: () =>
-              Navigator.of(context).pushNamed(Routes.ventilationInfographic),
-        )
-      ],
-      children: ventilationGuide
-          .map((e) => VentilationGuideContainer(guideSections: e.guideList))
-          .toList(),
-    );
+    final tabs = ventilationGuide.map((e) {
+      if (e is IntubationContent) {
```

Comment:
what does the if-else do if both branches return `e.heading` ?

Reply:
they're different types. 
Would you prefer `final tabs= ventilationGuide.map((dynamic e) => e.heading as String).toList();`? Might read less clearly, and prone to breaking with any refactor

Reply:
ah right thats tricky but I understand now.